src/pages/event/[id].tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Loader } from "../../components/Loader";
+import { useEffect, useState } from "react";
+import { useRouter } from "next/router";
+import type { Event } from "../../model/Event";
+
+const EventDetail: React.FC = () => {
+  const router = useRouter();
+  const { id } = router.query;
+  const [event, setEvent] = useState<Event>({} as Event);
```

Comment:
I see :)). It is semi legit. I would rather go this:
```suggestion
  const [event, setEvent] = useState<Event>();
```

I believe it will implicitly add `undefined` in the type.

And than later do `<p>Name: {event?.name}</p>`